app/mailers/user_organization_membership_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserOrganizationMembershipMailer < ApplicationMailer
+  def send_invite_email(user, organization, redirect_url, password)
```

Comment:
Rename this something like `new_user_invite`.  We may later have something different for existing users.